### ISSUE TITLE
Add a column for City Code?

### ISSUE THREAD

**Issue by nx01-a** [davidmegginson/ourairports-data#18]
Airports in larger metropolitan areas have an [alternative city code](https://en.wikipedia.org/wiki/IATA_airport_code#Cities_with_multiple_commercial_airports) that is supported by GDS engines. For example: ORD, MDW, and RFD are all under the Chicago city code of 'CHI'; LGA, JRK, EWR are NYC; HND & NRT are TYO. 

Some of these airports have the code in the Keywords section- but it would be helpful to have their own field in the dataset for just this purpose.

**Comment by davidmegginson** [comment]
Excellent suggestion -- thank you!

**Comment by galvesribeiro** [comment]
Hello! Was the City/Metropolitan code ever added? 

Thanks!

**Comment by kontrarie** [comment]
If this were implemented, would it have any added value outside the USA?

**Comment by galvesribeiro** [comment]
Yes, it would. Here in Brazil we also use Cities. I'm pretty sure many other countries do as well.

**Comment by nx01-a** [comment]
Yes, there are many of them ([see this page for examples](https://en.wikipedia.org/wiki/IATA_airport_code#Cities_with_multiple_commercial_airports)). Some cities of note: London, Paris, Seoul, Tokyo, Toronto, Buenos Aires,
